src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/IntegrationTestBase.kt

```diff
@@ -235,32 +229,26 @@ class IntegrationTestBase {
   internal fun checkEventLog(
     sourceSystemId: String,
     event: CPRLogEvents,
-    timeout: Long = 3,
     matchingEvents: (logEvents: List<EventLogEntity>) -> Unit,
   ) {
     awaitAssert(function = {
       matchingEvents(eventLogRepository.findAllByEventTypeAndSourceSystemIdOrderByEventTimestampDesc(event, sourceSystemId) ?: emptyList())
-    }, timeout)
+    })
   }
 
   internal fun checkEventLogByUUID(
     cluster: UUID,
     event: CPRLogEvents,
-    timeout: Long = 3,
     matchingEvents: (logEvents: List<EventLogEntity>) -> Unit,
   ) {
     awaitAssert(function = {
       matchingEvents(eventLogRepository.findAllByEventTypeAndPersonUUIDOrderByEventTimestampDesc(event, cluster) ?: emptyList())
-    }, timeout)
+    })
   }
 
-  private fun awaitAssert(function: () -> Unit, timeout: Long) = await atMost (Duration.ofSeconds(timeout)) untilAsserted function
-
-  internal fun awaitAssert(function: () -> Unit) = awaitAssert(function = function, timeout = 3)
+  internal fun awaitAssert(function: () -> Unit) = await atMost (Duration.ofSeconds(15)) untilAsserted function
```

Comment:
not sure this is a great idea, it might be annoying if a lot of tests fail
There's just one test which takes a while and needs a longer timeout